Assets/SEE/Game/Operator/EdgeOperator.cs

```diff
@@ -254,5 +271,148 @@ protected override void OnDisable()
             construction.KillAnimator();
             construction = null;
         }
+
+        /// <summary>
+        /// Implements a data flow visualization to indicate the direction of an edge.
+        /// </summary>
+        private class EdgeDataFlowVisualizer : MonoBehaviour
+        {
```

Comment:
Since this is a partial class, it should be a sibling to the original class in the file system. Hence, you can just put it in `Assets/SEE/Game/Operator/EdgeDataFlowVisualizer.cs`. 

(If, in the future, we are going to add a third partial class of the edge operator, we may want to create a separate directory for it, but this should be fine for now.)

Reply:
After renaming the class as suggested by @koschke the class went into `Assets/SEE/Game/Operator/EdgeDirectionVisualizer.cs`